tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -72,12 +37,56 @@ abs_path() {
     [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
 }
 
+
 # Exit after a failure
 die() {
     echo $@
     exit 1
 }
 
+# Uninstall existing pip packages
+uninstall_existing_pip_packages() {
```

Comment:
The uninstall sounds a bit dangerous. You would never need it if it was intended to run only in inside per build docker container. Because there would be nothing to uninstall in the first place. Also this does not allow concurrent builds outside docker (think mac os).

It would be great to think about any developer being able to use it on his/her computer. Without being afraid of messing up system. Hence no uninstall. Would it work if we would create per build virtualenv for python?

One can have working version of tensorflow installed on system and develop tensorflow on the same machine then.